kotlinx-coroutines-core/common/src/flow/operators/Delay.kt

```diff
@@ -64,37 +64,59 @@ fun main() = runBlocking {
  */
 @FlowPreview
 public fun <T> Flow<T>.debounce(timeoutMillis: Long): Flow<T> {
```

Comment:
Behaviour for zero timeout should be documented as well, especially because we are changing it